include/openrave/robot.h

```diff
@@ -1452,6 +1452,30 @@ class OPENRAVE_API RobotBase : public KinBody
     friend class Grabbed;
 };
 
+enum EnvironmentBodyRemoverRestoreOptions : uint8_t
+{
+    EBRRO_NoAbortOnInfoLost = 0, ///< will not abort even if any info cannot be restored.
```

Comment:
what about like this?
```suggestion
    EBRRO_NoAbortOnInfoLost = 0, ///< will not abort even if any info handled in this enum cannot be restored. Note that the remover would abort if there is an exception which is not handled in this enum.
```